### PR TITLE
Replace icon fonts with SVG

### PR DESCRIPTION
**Pull request by karimJWP**
### This PR will...
Expose rewind SVG in display and enable next in display

### Why is this Pull Request needed?
add back functionality that was lost while clearing merge conflicts

### Are there any points in the code the reviewer needs to double check?
n/a

### Are there any Pull Requests open in other repos which need to be merged with this?
view tests updated in https://github.com/jwplayer/jwplayer-commercial/pull/3940
https://github.com/jwplayer/jwplayer-plugin-related/pull/157

#### Addresses Issue(s):

JW8-90

### CODE REVIEW

**Comment by karimJWP** [comment]
test this please

File: src/js/view/controls/next-display-icon.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 import utils from 'utils/helpers';
+import UI from 'utils/ui';
 
 export default class NextDisplayIcon {
-    constructor(model) {
+    constructor(model, api) {
         const element = utils.createElement(displayIconTemplate('next', model.get('localization').next, NEXT_ICON));
 
+        new UI(element).on('click tap', function() {
+            api.next();
```
Comment (egreaves) [inline src/js/view/controls/next-display-icon.js]: `next` takes a `meta` argument, which by default is `{ reason: 'external' }`. In this case, it should be: `{ reason: 'interaction' }`

Comment (egreaves) [inline src/js/view/controls/next-display-icon.js]: Also, I noticed that `this.iconUI = ...` was removed. Is it no longer needed anywhere? If so, can we get rid of it from the rewind display icon as well?

Comment (karimJWP) [inline src/js/view/controls/next-display-icon.js]: @egreaves there is a playlistNext() api call that takes in a meta, https://github.com/jwplayer/jwplayer/blob/master/src/js/api/api.js#L748 ; next does not take one
the difference between the 2 is at the last item of a playlist, playlistNext will set item1, whereas next will start playing related

Comment (egreaves) [inline src/js/view/controls/next-display-icon.js]: @karimJWP good observation. I was looking at my local repo which is out of date.

**Comment by karimJWP** [comment]
test this please

**Comment by karimJWP** [comment]
@msiddique integration tests fail bc of 'Interface method initPlugin in namespace com.longtailvideo.jwplayer.plugins:IPlugin is implemented with an incompatible signature in class com.longtailvideo.jwplayer.ads.vast:VAST.' , is this specific to this PR ?

**Comment by karimJWP** [comment]
test this again

**Comment by robwalch** [comment]
@karimJWP The view test should show what the auto-start muted state looks like. It looks like it's not. This could be a change in the player/model state required to show it, or view logic that changes and is not being executed. I would expect to not see the controlbar, here and only see the unmute button:

screen shot 2017-08-16 at 2 27 23 pm

http://player-pr-test-jenkins.longtailvideo.com/builds/8965/archive/test/view/

**Comment by hghazzi** [comment]
@karimJWP Have the comments been addressed? Also, your branch needs a rebase

**Comment by karimJWP** [comment]
@robwalch updated view test. Will be available when commercial build completes
Note: we have a line in states.less that removes the button on paused. Looks weird to me but a comment was left stating 'Hide autostart mute button when paused by the browser'.
screen shot 2017-08-17 at 2 42 38 pm

**Comment by karimJWP** [comment]
@egreaves and @hghazzi  all comments addressed

**Comment by egreaves** [comment]
@karimJWP something's broken with the player controls. Take a look at any player.

**Comment by karimJWP** [comment]
@egreaves fixed

**Pull request merged**